docs/collect/system-metrics.md

```diff
@@ -43,9 +43,15 @@ First, [download windows_exporter](https://github.com/prometheus-community/windo
 with the following collectors enabled, changing `0.14.0` to the version you downloaded.
 
 ```powershell
-windows_exporter-0.14.0-amd64.exe --collectors.enabled="cpu,memory,net,logical_disk,os,system,logon,tcp"
+windows_exporter-0.14.0-amd64.exe --collectors.enabled="cpu,memory,net,logical_disk,os,system,logon,tcp,process"
 ```
 
+[process](https://github.com/prometheus-community/windows_exporter/blob/master/docs/collector.process.md) exposes
```

Comment:
> f the user doesn't use filtering flags - netdata monitoring can increase CPU usage significantly?

Yes.

---

Also, I think we shouldn't have this info in two places and just provide a link to https://github.com/netdata/go.d.plugin/tree/master/modules/wmi#requirements here

Reply:
No way for us to have some default filtering flags in place - and user can take them out if they want to?

Reply:
@shyamvalsan what would be the default? Independent of defined defaults, this will increase CPU usage anyway.

@ilyam8 we already had some windows explorer information at `netdata/netdata`, are you also suggesting to remove it? (ping @DShreve2 ).

Reply:
I suggest having a link to https://github.com/netdata/go.d.plugin/tree/master/modules/wmi#requirements instead of writing the same info.

Reply:
Updated.